rest/api/swagger.yaml

```diff
@@ -114,11 +114,13 @@ definitions:
               The count of master nodes requested in the cluster (must be > 0)
             type: integer
             format: int
+            default: 1
```

Comment:
Adding the default here is not only more explicit for the swagger API, it causes the code to generate with a *int64 instead of an int64 which is desired in this case